### PR TITLE
Network offering scopes

### PR DESCRIPTION
**Pull request by PaulAngus**
update for 4.13 to describe scoping per zone and per domain.

### CODE REVIEW

**Comment by PaulAngus** [comment]
request docbuild

**Comment by PaulAngus** [comment]
@shwstppr @nvazquez @anuragaw
can you review please.

**Comment by PaulAngus** [comment]
requesting docbuild

File: source/adminguide/networking.rst [hunk 1]
```diff
@@ -422,5 +421,20 @@ To create a network offering:
       this case, when you configure an egress rules for an isolated
       guest network, rules are added to allow the specified traffic.
 
+    - **Public**: Indicate whether the network offering should be available to
```
Comment (shwstppr) [inline source/adminguide/networking.rst]: Some formatting issue here?
Screenshot from 2019-07-17 13-03-15

**Comment by PaulAngus** [comment]
request docbuild

**Comment by PaulAngus** [comment]
request docbuild

File: source/adminguide/networking.rst (hunk 1 above)
Comment (PaulAngus) [reply source/adminguide/networking.rst]: formatting fixed

**Comment by andrijapanicsb** [comment]
LGTM

**Pull request merged**